### DISCUSSION TITLE
Plot Manager not starting any jobs

### DISCUSSION THREAD

**Discussion by cpereyra5199** [swar/Swar-Chia-Plot-Manager#1006] [Q&A]
Hello, I am having some trouble using the manager.  I have modified the config to use all defaults besides the directory changes needed, but the plotter doesn't seem to start any jobs. Any thoughts on why this could be happening?

Thanks!

**Comment by fd3adr32** [comment]
me too,
not starting any jobs, after kill chia.exe proc with taskmgr

**Reply by cpereyra5199** [reply]
My issue was actually related to pointing to the wrong file in the swar config for the chia.exe.
